Task 5/Array Helpers.js

```diff
@@ -0,0 +1,26 @@
+Array.prototype.square = function(){ return this.map( value => value*value ) };
+Array.prototype.cube = function(){ return this.map( value => value*value*value ) };
+Array.prototype.sum = function(){ 
+  let sum = 0;
+  this.forEach( value => sum += value );
+  return sum;
+};
+Array.prototype.average = function(){ 
+  let sum = 0, count = 0;
+  this.forEach( value => {
+    sum += value; 
+    count++; 
+  });
+  return sum/ count;
+};
+
+Array.prototype.even = function(){ 
+  let even = new Array();
+  this.map( value => !(value%2) && even.push(value));
+  return even;
+};
+Array.prototype.odd = function(){ 
+  let odd = new Array();
+  this.map( value => (value%2) && odd.push(value));
+  return odd; 
```

Comment:
While these approaches work, I would recommend you to try using the `array-helper` methods. Use `map, reduce, filter` methods. All these functions will have a single return statement if used the above helper methods

MDN References:
1. `map`: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map

2. `reduce`: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/Reduce

3. `filter`: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/Filter